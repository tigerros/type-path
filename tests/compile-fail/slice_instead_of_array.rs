use type_path::type_path;

const ITER_REPEAT_PATH: &[&str; 4] = type_path!(::std::iter::repeat);

fn main() {}
