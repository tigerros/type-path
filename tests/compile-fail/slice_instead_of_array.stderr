error[E0308]: mismatched types
 --> tests/compile-fail/slice_instead_of_array.rs:3:38
  |
3 | const ITER_REPEAT_PATH: &[&str; 4] = type_path!(::std::iter::repeat);
  |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `&[&str; 4]`, found `[&str; 4]`
  |
  = note: this error originates in the macro `type_path` (in Nightly builds, run with -Z macro-backtrace for more info)
